docs/installation.md

```diff
@@ -50,17 +50,30 @@ list of instruments:
         "platformType": "ILLUMINA",
         "name": "default",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/run1"
+        ]
       },
       {
         "path": "/srv/sequencer/promethion",
         "platformType": "OXFORDNANOPORE",
         "name": "promethion",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/test/run1",
+            "/run2"
+        ]
       }
     ]
 
+The ignoreSubdirectories field is used to exclude subdirectories from processing. For example, a 
+known bad run or a test folder holding temporary files like /test/mytestrun. This is optional 
+and in cases where ignoreSubdirectories is not included in the configuration file, all 
+subdirectories under that instrument path will be processed by Run Scanner. For some examples of 
+how to use this feature, check out the tests inside `SchedulerTest`.
```

Comment:
The people installing and running Run Scanner are often not Java developers. Nowhere else in the docs do we recommend looking at source code for explanations of features. (And also, unit tests are in general not a good way to understand features because of the outsized focus on edge cases)